.github/workflows/release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           node-version: 18
           registry-url: https://registry.npmjs.org/
       - run: npm i
-      - run: npm run test:all
+      - run: npm test -w db-service -w sqlite -w postgres -- --maxWorkers=1
```

Comment:
once hana is also released it should be possible to replace it with:
`npm test -ws -- --maxWorkers=1`